src/Tools/dotnet-counters/Exporters/ConsoleWriter.cs

```diff
@@ -66,6 +66,12 @@ public ObservedTagSet(string tags)
         private string _errorText = null;
 
         private int maxRow = -1;
+        private bool useAnsi = false;
+
+        public ConsoleWriter(bool _useAnsi) 
+        {
+            this.useAnsi = _useAnsi;
+        }
```

Comment:
```suggestion
        private bool useAnsi = false;

        public ConsoleWriter(bool useAnsi) 
        {
            this.useAnsi = useAnsi;
        }
```

nit: Usually prefacing with underscore is for member fields and not parameters.